src/main/kotlin/me/zeroeightsix/kami/BaritoneIntegration.kt

```diff
@@ -13,8 +13,7 @@ import me.zeroeightsix.kami.setting.KamiConfig
 import me.zeroeightsix.kami.setting.settingInterface
 
 object BaritoneIntegration {
-
-    private val present by lazy {
+    val present by lazy {
         try {
```

Comment:
This was private for a reason. Look at the `invoke` overload on BaritoneIntegration, you're supposed to check its value like so:

```kotlin
BaritoneIntegration {
    // This only runs if `present` is true!
}
```